src/ttsnake.c

```diff
@@ -722,6 +730,17 @@ void * userinput ()
       case 'r':
         restart_game = 1;	/* Restart game. */
       break;
+      case 'p':
+        if (pause_game) {
+           /* set beginning to current time and resume game */ 
+          gettimeofday (&beginning, NULL);
```

Comment:
Since `beginning` now doesn't store the time when the game started, we should consider changing the variable name